EtlPipelineBlobTrigger/course_stats.py

```diff
@@ -129,10 +187,58 @@ def get_raw_continuation_list(raw_course_data):
         return continuation_list
 
 
-class CourseStats:
-    def get_stats(self, raw_course_data):
-        stats = {}
+class Employment:
+    """Extracts and transforms the Employment course element"""
 
-        continuation = Continuation()
-        stats['continuation'] = continuation.get_continuation(raw_course_data)
-        return stats
+    def __init__(self):
+        self.shared_utils = SharedUtils()
+
+    def get_key(self, xml_key):
+        return {
+            "EMPUNAVAILREASON": "unavailable",
+            'EMPPOP': 'number_of_students',
+            'EMPRESP_RATE': 'response_rate',
+            "EMPAGG": "aggregation_level",
+            "EMPSBJ": "subject",
+            'WORKSTUDY': 'in_work_or_study',
+            "STUDY": 'in_study',
+            "ASSUNEMP": 'assumed_to_be_unemeployed',
+            "BOTH": 'in_work_and_study',
+            "NOAVAIL": 'not_available_for_work_or_study',
+            "WORK": 'in_work',
+        }[xml_key]
+
+    def get_emp_unavailable(self, xml_elem):
+        subj_key = 'EMPSBJ'
+        agg_key = 'EMPAGG'
+        unavail_reason_key = 'EMPUNAVAILREASON'
+        return self.shared_utils.get_unavailable(xml_elem, subj_key, agg_key,
+                                                 unavail_reason_key)
+
+    def get_employment(self, raw_course_data):
+        subj_key = 'EMPSBJ'
+        agg_key = 'EMPAGG'
+        unavail_reason_key = 'EMPUNAVAILREASON'
+        element_key = 'EMPLOYMENT'
+
+        json_elem_list = []
```

Comment:
This structure can be reused for get_continuation?

Reply:
Yes, as discussed planning on more refactoring on next story.

Reply:
Good good